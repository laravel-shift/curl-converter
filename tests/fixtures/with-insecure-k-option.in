curl -H 'Accept: application/json' https://example.com -k
