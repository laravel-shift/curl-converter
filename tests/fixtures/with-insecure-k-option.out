Http::acceptJson()
    ->withoutVerifying()
    ->get('https://example.com');
